src/test/java/org/elasticsearch/indices/store/IndicesStoreIntegrationTests.java

```diff
@@ -95,6 +95,12 @@ public void indexCleanup() throws Exception {
         assertThat(Files.exists(indexDirectory(node_3, "test")), equalTo(false));
 
         logger.info("--> move shard from node_1 to node_3, and wait for relocation to finish");
+        SlowClusterStateProcessing disruption = null;
+        if (randomBoolean()) {
+            disruption = new SlowClusterStateProcessing(node_3, getRandom(), 0, 0, 1000, 2000);
```

Comment:
do we need random sleeps here or can we use BlockClusterStateProcessing to make this more reproducible?